src/rewritable_units/element.rs

```diff
@@ -45,7 +45,7 @@ pub struct Element<'r, 't> {
     start_tag: &'r mut StartTag<'t>,
     end_tag_mutations: Option<Mutations>,
     modified_end_tag_name: Option<Bytes<'static>>,
-    end_tag_handler: Option<EndTagHandler<'static>>,
```

Comment:
Does it matter that we're getting rid of the `Option` here? What about `Vec<Option<EndTagHandler<'static>>>`?

Reply:
The Option here was just to make a container of 0 or 1 elements.  Using a `Vec<_>` we generalize to any number of elements.